source-jira/tests/snapshots/test_snapshots/test_discover/capture.stdout.json

```diff
@@ -0,0 +1 @@
 customer user)", 'readOnly': True}}}, 'comment': {'type': 'object', 'description': 'A comment about the worklog in [Atlassian Document Format](https://developer.atlassian.com/cloud/jira/platform/apis/document/structure/). Optional when creating or updating a worklog.'}, 'created': {'type': 'string', 'description': 'The datetime on which the worklog was created.', 'format': 'date-time', 'readOnly': True}, 'updated': {'type': 'string', 'description': 'The datetime on which the worklog was last updated.', 'format': 'date-time', 'readOnly': True}, 'visibility': {'description': 'Details about any restrictions in the visibility of the worklog. Optional when creating or updating a worklog.', 'type': 'object', 'properties': {'type': {'type': 'string', 'description': 'Whether visibility of this item is restricted to a group or role.', 'enum': ['group', 'role']}, 'value': {'type': 'string', 'description': 'The name of the group or role to which visibility of this item is restricted.'}}}, 'started': {'type': 'string', 'description': 'The datetime on which the worklog effort was started. Required when creating a worklog. Optional when updating a worklog.', 'format': 'date-time'}, 'timeSpent': {'type': 'string', 'description': "The time spent working on the issue as days (\\#d), hours (\\#h), or minutes (\\#m or \\#). Required when creating a worklog if `timeSpentSeconds` isn't provided. Optional when updating a worklog. Cannot be provided if `timeSpentSecond` is provided."}, 'timeSpentSeconds': {'type': 'integer', 'description': "The time in seconds spent working on the issue. Required when creating a worklog if `timeSpent` isn't provided. Optional when updating a worklog. Cannot be provided if `timeSpent` is provided."}, 'id': {'type': 'string', 'description': 'The ID of the worklog record.', 'readOnly': True}, 'issueId': {'type': 'string', 'description': 'The ID of the issue this worklog is for.', 'readOnly': True}, 'properties': {'type': 'array', 'description': 'Details of properties for the worklog. Optional when creating or updating a worklog.', 'items': {'type': 'object', 'properties': {'key': {'type': 'string', 'description': 'The key of the property. Required on create and update.'}, 'value': {'description': 'The value of the property. Required on create and update.'}}}}, '_meta': {'type': 'object', 'properties': {'row_id': {'type': 'integer'}}, 'required': ['row_id']}}, 'additionalProperties': True, 'description': 'Details of a worklog.', 'x-infer-schema': True}, 'key': ['/id']}, {'recommendedName': 'jira_settings', 'resourceConfig': {'stream': 'jira_settings', 'syncMode': 'full_refresh'}, 'documentSchema': {'$schema': 'http://json-schema.org/draft-07/schema#', 'type': 'object', 'properties': {'id': {'type': 'string', 'description': 'The ID of the application property. The ID and key are the same.'}, 'key': {'type': 'string', 'description': 'The key of the application property. The ID and key are the same.'}, 'value': {'type': 'string', 'descriptio
```

Comment:
These snapshot outputs need to follow the canonical structure (see [source-asana, for example](https://github.com/estuary/connectors/tree/main/source-asana/tests/snapshots)) rather than each being in a subdirectory, and should also be formatted to be human-readable, that way we can easily see and understand if there are any differences from subsequent changes.